### PR TITLE
ref(eap): Add configuration for attributes meta

### PR DESCRIPTION
**Pull request by evanh**
Add the storage for the spans attributes meta. Also add a basic test file for the eap dataset, and a test for the spans storage and the meta storage.

Depends on https://github.com/getsentry/snuba/pull/6152

### CODE REVIEW

File: snuba/datasets/configuration/events_analytics_platform/storages/spans_attributes_meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  dist_table_name: spans_attributes_meta_dist
+  partition_format: [date]
+allocation_policies:
+  - name: ConcurrentRateLimitAllocationPolicy
+    args:
+      required_tenant_types:
+        - organization_id
+        - referrer
+      default_config_overrides:
+        is_enforced: 0
```
Comment (volokluev) [inline snuba/datasets/configuration/events_analytics_platform/storages/spans_attributes_meta.yaml:32]:
```suggestion
      default_config_overrides:
        is_enforced: 0
```

File: snuba/datasets/configuration/events_analytics_platform/storages/spans_attributes_meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  - name: ReferrerGuardRailPolicy
+    args:
+      required_tenant_types:
+        - referrer
+      default_config_overrides:
+        is_enforced: 0
+        is_active: 0
```
Comment (volokluev) [inline snuba/datasets/configuration/events_analytics_platform/storages/spans_attributes_meta.yaml:39]:
```suggestion
```

File: snuba/datasets/configuration/events_analytics_platform/storages/spans_attributes_meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  - name: BytesScannedRejectingPolicy
+    args:
+      required_tenant_types:
+        - organization_id
+        - referrer
+      default_config_overrides:
+        is_active: 0
+        is_enforced: 0
```
Comment (volokluev) [inline snuba/datasets/configuration/events_analytics_platform/storages/spans_attributes_meta.yaml:47]:
```suggestion
```

**Review by volokluev — APPROVED** [review]

**Comment by volokluev** [comment]
Didn't we remove this because the load on the cluster was too high?

**Comment by evanh** [comment]
This was added separately.

**Pull request closed without merging**